### PR TITLE
Adding graffiti hex check and conversion to string

### PR DESCRIPTION
**Pull request by patnir**
For context: https://iflabs.slack.com/archives/C02PF56HESK/p1696291835711159 

image

### CODE REVIEW

File: components/ChainTree/BlockPreview.tsx [hunk 1]
```diff
@@ -32,7 +33,9 @@ const PREVIEW_BLOCKS = [
   },
   {
     label: 'Graffiti',
-    value: safeProp('graffiti'),
```
Comment (danield9tqh) [inline components/ChainTree/BlockPreview.tsx:35]: Why do we have to render the `Box` here now?

File: pages/blocks/[id].tsx [hunk 2]
```diff
@@ -86,7 +85,9 @@ const BLOCK_CARDS = [
   {
     key: 'graffiti-card',
     label: 'Graffiti',
-    value: safeProp('graffiti'),
+    value: pipe(safeProp('graffiti'), graffiti => {
```
Comment (danield9tqh) [inline pages/blocks/[id].tsx:88]: Same

File: components/ChainTree/BlockPreview.tsx (hunk 1 above)
Comment (patnir) [reply components/ChainTree/BlockPreview.tsx:35]: Removed

File: pages/blocks/[id].tsx (hunk 2 above)
Comment (patnir) [reply pages/blocks/[id].tsx:88]: done

**Review by danield9tqh — APPROVED** [review]

**Pull request merged**